Controllers/CardController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using GemManager.Models;
+using GemManager.Repositories;
+
+namespace GemManager.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CardController : ControllerBase
+    {
+        private readonly IGemRepository _gemRepository;
+        private readonly IRepository<User> _userRepository;
+
+        public CardController(IGemRepository gemRepository, IRepository<User> userRepository)
```

Comment:
The set-up looks good but maybe we can inject the repos into the mediator handlers instead? CardController can just concentrate on card stuff and send requests to the mediator to do actions with the gems and users.

Reply:
I totally agree with that indeed. This was a very first setup that the controller works with the injections. But then all repos would be available only through mediatr handlers indeed.